plugins/containers/plugin/languages/en.yml

```diff
@@ -38,34 +38,49 @@ en:
       desc: "A small cardboard box."
     trash_bin:
       title: Trash Can
-      desc: "A regular trash bin. Its bottom is coated in a layer of sticky manga pages."
+      desc: "A small trash bin that contains various types of trash from many people."
     dumpster:
       title: Trash Dumpster
-      desc: "A large dumpster used to hold significant amounts of garbage. It smells awful."
+      desc: "A large dumpster used to hold significant amounts of garbage that reeks of various disgusting smells."
     wooden_crate:
       title: Crate
-      desc: "A wooden crate."
+      desc: "A simple wooden crate used to hold various objects."
     big_wooden_crate:
       title: Big Crate
-      desc: "A large wooden crate."
+      desc: "A larger wooden crate used for transportation of items."
     file_cabinet:
       title: Filing Cabinet
-      desc: "A rusty filing cabinet."
+      desc: "A rusty filing cabinet that consists of two large drawers."
     tall_file_cabinet:
       title: Tall Filing Cabinet
-      desc: "A large rusty filing cabinet."
+      desc: "A large rusty filing cabinet that consists of many drawers."
     storage_closet:
       title: Closet
-      desc: "A closet used for holding clothes."
+      desc: "A large metallic closet that is used to store not only clothes, but also any type of item that can fit in it."
     large_fridge:
       title: Industrial Fridge
-      desc: "A large refrigerator. It looks old and overused."
+      desc: "An Industrial-Grade fridge used to store many food items of various types."
     counter:
       title: Counter
-      desc: "A counter piece."
+      desc: "A large, metallic counter with a large place able to be utilized for storage."
```

Comment:
Replace `metallic` with `metal`, remove comma. Actually just replace it with `A large metal counter commonly used to store money.`